### ISSUE TITLE
Add option for auto theme from device settings

### ISSUE THREAD

**Issue by sgshivamgarg8** [Skyost/DayNightSwitcher#4]
Add options for taking theme from system settings, i.e. Auto theme

**Comment by Skyost** [comment]
You can use `MediaQuery.platformBrightnessOf(context)` to get system's brightness.

**Comment by sgshivamgarg8** [comment]
Yes I know that but I was thinking if you could implement that somehow in your widget only.
Although I am not quiet sure of how the UI will look in that case

**Comment by Skyost** [comment]
I think it's a bit out of purpose for this plugin. But I've already implemented something like that in [one of my apps](https://github.com/Skyost/Bacomathiques/blob/app/lib/app/dialogs/theme_mode.dart#L47).
